test/testhelpers.sh

```diff
@@ -161,6 +161,21 @@ setup() {
   }
   cp "$HOME/.gitconfig" "$HOME/.gitconfig-backup"
 
+  if [[ `git config --system credential.helper | grep osxkeychain` == "osxkeychain" ]]
```

Comment:
The osxkeychain doesn't take any arguments. The extra quotes don't seem necessary?